docs/src/test-components-js.md

```diff
@@ -61,6 +62,14 @@ yarn create playwright --ct
 ```
 
 </TabItem>
+
+<TabItem value="pnpm">
+
+ ```bash
+ pnpm dlx create-playwright --ct
```

Comment:
Unlike npx, `pnpm dlx` won't be using the installed version of the package, will always be getting the newest one. While it works great in this use case, the rest of the documentation with `pnpm dlx playwright install` instead of `npx playwright install` will be completely broken.

Reply:
on, so I learned that `pnpx exec` does a fine job with install, while dlx does a fine job with create-playwright.

Reply:
https://github.com/microsoft/create-playwright/pull/76/files has landed, we need to push create-playwright and all is good.

Reply:
Ahh alright, nice!